docs/source/user-guide/ui-api-guide/external-datasets/managing.md

```diff
@@ -1,27 +1,36 @@
+External datasets contain profiles that are quite similar to the profiles produced by a Merlin simulation. They describe the behavior of some dynamic characteristic over time and can be viewed in the Aerie UI along with a plan. External profiles come from the user, rather than Merlin's simulation capabilities, and can be uploaded at any time before or after simulation.
 
-Ultimately, the purpose of external profiles is up to the user. You might be looking to include some geometry information to assist in building a plan, or you may simply be adding power and thermal modeling results to be viewable with existing simulation results. Just be aware that at this time external profiles are not accessible to the mission model during simulation, and are simply for viewing purposes in the UI.
+Ultimately the purpose of external datasets is up to the user. You might be looking to include some geometry information to assist in building a plan, or you may simply be adding power and thermal modeling results to be viewable with existing simulation results. Just be aware that at this time external profiles are not accessible to the mission model during simulation, and are simply for viewing purposes in the UI.
 
-## How To Upload A Dataset
-To upload external profiles, you will need to add them to a specific plan. A collection of profiles that all start at the same time are called an "External Dataset". The dataset must be associated with a plan, and will persist as long as the plan exists, or until it is deleted by a user. An external dataset can be added via the GraphQL mutation `addExternalDataset`.
+## Add External Dataset Mutation
 
-### GraphQL Mutation: `addExternalDataset`
-The `addExternalDataset` GraphQL mutation takes three parameters as specified below:
+To upload an external dataset you need to add it to a specific plan. The dataset will persist as long as the plan exists, or until it is explicitly deleted. An external dataset can be added via the GraphQL mutation `addExternalDataset`:
 
-| Parameter      | Type    | Description                                                                      |
-|----------------|---------|----------------------------------------------------------------------------------|
-| `planId`       | Integer | The ID of the plan to associate the external dataset with.                       |
-| `datasetStart` | String  | The UTC timestamp the dataset is based from.<br/>UTC Format: `yyyy-dddThh:mm:ss` |
-| `profileSet`   | Object  | The set of precomputed profiles that make up the external dataset.               |
+```
+mutation AddExternalDataset($planId: Int!, $datasetStart: String!, $profileSet: ProfileSet!) {
```

Comment:
This mutation and `DeleteExternalDataset` [already existed in `api-examples.rst`](https://github.com/NASA-AMMOS/aerie/blob/develop/docs/source/user-guide/ui-api-guide/api-examples.rst#add-an-external-dataset). Instead of deleting it from that file, could we instead update the mutation defined there and use an `include` directive?

Reply:
No I intentionally removed it in this case. I don't think we need it in both places.

Reply:
I thought `api-examples.rst` was a reference page containing all of our non-example API queries (`non-example` meaning 'not tailored to a specific example', like the mutations originally under the `Example 1/2/3` headers on this page). If this is a mistaken understanding then could you clarify what `api-examples.rst` is supposed to contain?

Reply:
Previously our `api-examples` page existed to show queries we did not have explained elsewhere (or some different way to show a query we had elsewhere). In this case these queries are explained in this doc, and I don't think there is much utility in having them in `api-examples` without the context. When users ask about how to add external profiles in Aerie we should point them to this page and not the API examples page.

Reply:
Right, but when we finish the documentation I don't know if there's any queries currently on `api-examples` that wouldn't also be on a explanation page. This is where my understanding of a "quick syntax reference page" comes from. If we're concerned about users needing the context, however, we can link to the appropriate page per header, along with adding a statement at the top of the page along the lines of "This page only contains example GraphQL queries used with Aerie. For an explanation of how each query works and when to use it, see the corresponding documentation." We could also rename the page to better reflect this change in expected use.

Additionally, wouldn't having the queries all hosted in one spot make it easier for us as developers to update the queries (especially if we link to the explanation page)?

Reply:
All good points @Mythicaeda. Let's discuss more next week.